packages/dd-trace/src/appsec/reporter.js

```diff
@@ -162,9 +162,27 @@ function reportSchemas (derivatives) {
   if (!rootSpan) return
 
   const tags = {}
-  for (const [address, value] of Object.entries(derivatives)) {
+  for (const [tag, value] of Object.entries(derivatives)) {
+    if (isFingerprintDerivative(tag)) continue
     const gzippedValue = zlib.gzipSync(JSON.stringify(value))
-    tags[address] = gzippedValue.toString('base64')
+    tags[tag] = gzippedValue.toString('base64')
+  }
```

Comment:
It's a lot of code duplication between reportSchemas() and reportFingerprints() no ? :/
```suggestion
    if (isFingerprintDerivative(tag)) {
      const gzippedValue = zlib.gzipSync(JSON.stringify(value))
      value = gzippedValue.toString('base64')
    }

    tags[tag] = value
  }
```

Reply:
(if the suggestion above isn't clear, i mean combine both functions in one)